src/plugins/console/public/plugin.ts

```diff
@@ -46,23 +43,32 @@ export class ConsoleUIPlugin implements Plugin<void, void, AppSetupUIPluginDepen
       category: FeatureCatalogueCategory.ADMIN,
     });
 
-    devTools.register({
+    return devTools.register({
```

Comment:
The original use of `async` meant that the return value of calling `devTools.register` was published as this `setup` method's contract.

Reply:
Not 100% sure I follow 😅, how did `async` cause the call to `devTools.register` to be the return value of `setup`? AFAICT the return value of `async setup` was `Promise<void>`.

In fact, I don't think we should be returning the value of devTools.register as the public contract. Not sure it makes sense to expose our registration to dev tools to other plugins, thus giving them a way to enable and disable console?

Reply:
To clarify, I think the return value of setup should go from `Promise<void>` to just `void` which should be achieved by just removing `async` and not returning anything.

Reply:
You're right! I confused the implicit return of a promise with implicitly returning a value. Thanks for pointing this out!